### ISSUE TITLE
is this abandoned? plugin not working in matomo 3.10

### ISSUE THREAD

**Issue by pedro-nonfree** [hoamer/IceCastStatistics#6]
here a matomo ~~3.10~~ 3.11 instance

icecast-statistics plugin is not working

![2019-10-03-141903_641x499_scrot](https://user-images.githubusercontent.com/39959198/66126425-ad59e080-e5e9-11e9-8f13-15b9e9e93113.png)

cannot save custom variables of password and stream

related to #3 how can I see the warnings? I tried `tail -f /var/log/apache2/error.log` or `tail -f /var/log/icecast2/error.log` and I see nothing related to this plugin, I don't know how to troubleshoot this

my hopes are void: this git repository never solved an issue and abandoned since 2017. Maybe we have to point to another repository

would be very happy to see more people alive here

**Comment by matias-tecnosoul** [comment]
Any news regarding this?

**Comment by matias-tecnosoul** [comment]
what do you reckon @pedro-nonfree  ?? should we try to fork it? it'd be really interesting to have smthg like this again...
Or is it working for anyone??

**Comment by valerio-bozzolan** [comment]
Precisely I obtain this error:

```
Class 'Piwik\Plugins\IceCastStatistics\SystemSettings' not found
in /home/www-data/piwik/plugins/IcecastStatistics/Widgets/GetIceCastStatistics.php line 74 
```

Visiting `/index.php?module=IcecastStatistics&action=getIceCastStatistics` on Matomo 3.9.1.

**Comment by pedro-nonfree** [comment]
@matias-tecnosoul yes, I think we should fork it, but first let's solve the problem

@valerio-bozzolan niiiiice, last time I tried could not get the error, with that link, getting the error messages is good to understand how to fix it

let's see if I find some time to work on this
